compiler-rt/lib/tsan/rtl/tsan_interceptors_posix.cpp

```diff
@@ -3135,7 +3139,7 @@ void InitializeInterceptors() {
 
   TSAN_INTERCEPT(fork);
   TSAN_INTERCEPT(vfork);
-#if SANITIZER_LINUX
+#if SANITIZER_LINUX && !SANITIZER_ANDROID
```

Comment:
@fmayer --- what's the compatibility expectation for compiler-rt? i'm wondering whether i should make pthread_create() and fork() call clone() via a hidden alias so they don't get intercepted ... but but then there's nothing to do here other than wait a decade?